File: latihan b/Latihan2 Py/Produk.py
class Produk():
#kelas yang digunakan untuk mengimplementasikan Produk
#atribut Produk
	

	price = 0		#price
	idproduk = ""	#idProduk
	
	def __init__(self, price = 0, idproduk = ""):
	#costructor
		self.price = price
		self.idproduk = idproduk
	
	#mengeset nilai price
	def setprice(self, price):
		self.price = price
        
	#mengeset nilai idProduk
	def setidproduk(self, idproduk):
		self.idproduk = idproduk

	#mengembalikan nilai price
	def getprice(self):
		return self.price

	#mengembalikan nilai idProduk
	def getidproduk(self):
		return self.idproduk
	
	

File: latihan a/produk.java
class produk
{ 
	/*kelas yang digunakan untuk mengimplementasikan hardware/
	/*atribut produk*/
		private int price;			//price
		private String idproduk;	//idproduk 
	
		public produk(){
		/*costructor*/
		}
		
		public produk(int a, String b){
		/*costructor*/
			this.price = a;
			this.idproduk = b;
		}
	
	/*mengeset nilai price*/
	public void setprice(int a)
	{
		this.price = a;
	}
	/*mengeset nilai idproduk*/
	public void setidproduk(String b)
	{
		this.idproduk = b;
	}

	/*mengembalikan nilai price*/
	int getprice()
	{
		return this.price;
	}
	/*mengembalikan nilai idproduk*/
	String getidproduk()
	{
		return this.idproduk;
	}
}